addons/l10n_be_hr_payroll_fleet/models/fleet.py

```diff
@@ -111,15 +115,15 @@ def _get_car_atn(self, acquisition_date, car_value, fuel_type, co2):
                 atn = 0.0
             else:
                 if fuel_type in ['diesel', 'hybrid']:
-                    reference = 87.0
+                    reference = 86.0
```

Comment:
@tivisse We should probably update those value according to 2019 and maybe backport them in 10, 11 and 12